cots/eat_cwtclaims.go

```diff
@@ -0,0 +1,86 @@
+package cots
```

Comment:
This is not the right place (CoRIM/cots) to introduce EAT CWT Claims.

Please raise a github issue to remove this from CoRIM